src/app/action-items/action-items.component.spec.ts

```diff
@@ -1,21 +1,52 @@
+import { MatDialogRef } from '@angular/material/dialog';
+import { AddItemComponent } from './../add-item/add-item.component';
 import { DaysLeftToDeadlineService } from './../services/daysLeftToDeadlineService/days-left-to-deadline.service';
 import { async, ComponentFixture, TestBed, tick, fakeAsync } from '@angular/core/testing';
 import { of } from 'rxjs';
-import { TasksService, ActionTasksElement } from '../services/tasksService/tasks.service';
-import { ActionItemsComponent, ActionTasksElementMapped } from './action-items.component';
+import { TasksService, ActionItem } from '../services/tasksService/tasks.service';
+import { ActionItemsComponent, ActionItemMapped } from './action-items.component';
 import { CircleColorPipe } from '../pipes/circleColorPipe/circle-color.pipe';
+import {
+  MatDialog,
+  MatDialogConfig,
+  MatInputModule,
+  MatSelectModule,
+  MatDatepickerModule,
+  MatDialogModule,
+  MatNativeDateModule
+} from '@angular/material';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { By } from '@angular/platform-browser';
 
 describe('ActionItemsComponent', () => {
   let component: ActionItemsComponent;
   let fixture: ComponentFixture<ActionItemsComponent>;
   let tasksService: TasksService;
   let daysLeftToDeadlineService: DaysLeftToDeadlineService;
   let element;
+  let dialog: MatDialog;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ActionItemsComponent, CircleColorPipe],
-      providers: [TasksService]
+      declarations: [ActionItemsComponent, CircleColorPipe, AddItemComponent],
+      imports: [
+        FormsModule,
+        ReactiveFormsModule,
+        MatInputModule,
```

Comment:
Could you add `MaterialModule` for all modules from material library and use it across the app?

Reply:
This one is still not addressed.